pages/login-signup/signup.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/pages/login-signup/login.css">
+
+    <title>Organix</title>
+</head>
+
+<body>
+    <nav class="nav fixed bg-light box-shadow">
+        <a class="brand">Organix </a>
+        <ul class="navbar-nav">
+            <!-- <li class="nav-link"><a href="" class="nav-link">SHOP</a></li> -->
+        </ul>
+        <ul class="navbar-nav">
+            <li class="nav-link"><a href="/pages/login-signup/login.html" class="nav-link">LOGIN</a></li>
+            <li class="nav-link"><a href="/pages/login-signup/signup.html" class="nav-link">SIGNUP</a></li>
+            <li class=""><input type="text" class="input-f" placeholder="Search"><i class="fa-solid fa-apple-whole"></i>
+            </li>
+        </ul>
+        <button class="btn-badge">
+            <i class="fas fa-shopping-cart"></i>
+            <span class="count">5</span>
+        </button>
+        <button class="btn-badge">
+            <i class="fas fa-heart"></i>           
+         <span class="count">5</span>
+        </button>
+    </nav>
+
+    <div class="container">
+        <div class="items">
+            <div class="card card-login">
+                <div class=" card-body description">
+                    <label for="">Name<input type="text" class="input-field outline" required></label>
+                    <label for="">Email<input type="email" class="input-field outline"required></label>
+                    <label for="">Password<input type="password" class="input-field outline"required></label>
+                    <label for="">Confirm Password<input type="password" class="input-field outline"required></label>
```

Comment:
Please name password and confirm password right aligned as it not aligning with name and email